dev/src/GitHub.php

```diff
@@ -290,4 +291,37 @@ private function getRepo($target)
 
         return $res;
     }
+
+    /**
+     * Add webhook
+     */
+    public function addWebhook(
+        string $target,
+        string $webhookUrl,
+        string $secret
+    ) {
+        try {
+            $res = $this->client->post(sprintf(
+                self::GITHUB_WEBHOOK_CREATE_ENDPOINT,
+                $this->cleanTarget($target)
+            ), [
+                'auth' => [null, $this->token],
+                'json' => [
+                    'name' => 'web',
+                    'active' => true,
+                    'events' => ['push'],
+                    'config' =>  [
+                        'url' => $webhookUrl,
+                        'content_type' => 'json',
+                        'secret' => $secret,
+                        'insecure_ssl' => false,
+                    ],
+                ],
+            ]);
+
+            return $res->getStatusCode() === 201;
+        } catch (\Exception $e) {
+            return null;
```

Comment:
Worth at least logging the exception maybe?

Reply:
Good call. done!